File: fizz-buzz.cpp
#include <iostream>

using namespace std;

int main()
{
    int N;
    cin >> N;
    for (int i = 1; i <= N; i++)
    {
        if (i % 15 == 0)
        {
            cout << "Fizz Buzz" << endl;
            continue;
        }

        if (i % 3 == 0)
        {
            cout << "Fizz" << endl;
            continue;
        }
        else if (i % 5 == 0)
        {
            cout << "Buzz" << endl;
            continue;
        }

        cout << i << endl;
    }
}

File: ruby_code/fizz_buzz.rb
n = gets.to_i
n += 1
n.times do |i|
    if i == 0
        next
    end
    
    if i % 3 == 0 && i % 5 == 0
        puts "Fizz Buzz"
    elsif i % 3 == 0 && i % 5 != 0
        puts "Fizz"
    elsif i % 3 != 0 && i % 5 == 0
        puts "Buzz"
    else
        puts i
    end
end